make/RunTests.gmk

```diff
@@ -45,7 +45,7 @@ ifneq ($(TEST_VM_OPTS), )
 endif
 
 $(eval $(call ParseKeywordVariable, TEST_OPTS, \
-    SINGLE_KEYWORDS := JOBS TIMEOUT_FACTOR JCOV JCOV_DIFF_CHANGESET, \
+    SINGLE_KEYWORDS := JOBS TIMEOUT_FACTOR JCOV JCOV_DIFF_CHANGESET REPORT, \
```

Comment:
I think we only add options that really are common to more than one test framework to TEST_OPTS.

Reply:
Right, good catch. This one is unnecessary. Fixed in new commit.